be/src/storage/lake/tablet_manager.cpp

```diff
@@ -473,6 +473,15 @@ StatusOr<TabletSchemaPtr> TabletManager::get_tablet_schema(int64_t tablet_id, in
         if (version_hint != nullptr) {
             *version_hint = metadata->version();
         }
+
+        // reload data size
+        int64_t size = 0;
+        for (const auto& rowset : metadata->rowsets()) {
+            size += rowset.data_size();
+        }
+        std::unique_lock wrlock(_meta_lock);
+        _tablet_in_writing_size[tablet_id] = size;
+        VLOG(1) << "tablet " << tablet_id << " reload in writing data size: " << _tablet_in_writing_size[tablet_id];
```

Comment:
The code branch here should be effectively unreachable.

Reply:
not really understand what you mean

Reply:
IMO, the codes you added here is useless.